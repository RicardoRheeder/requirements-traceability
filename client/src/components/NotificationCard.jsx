import React from 'react'

export default function NotificationCard({ notification }) {
  const style = {
    color: 'blue',
    fontSize: '20px',
  }
  return <div style={style}>{notification}</div>
}
