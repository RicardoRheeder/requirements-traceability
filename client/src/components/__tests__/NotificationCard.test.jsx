import React from "react"
import Enzyme, { shallow, mount} from "enzyme"
import NotificationCard from "../NotificationCard"
import Adapter from "enzyme-adapter-react-16"
import { Provider } from "react-redux";


Enzyme.configure({adapter: new Adapter() });

describe("NotificationCard", () => {
    
    test("renders", () => {
        const wrapper = shallow(<NotificationCard notification = "test"/>);
        expect(wrapper.exists()).toBe(true);
        expect(wrapper).toMatchSnapshot();
    });
    test("renders with children", () => {
        const wrapper = mount(<NotificationCard notification = "test"/>);
        expect(wrapper.exists()).toBe(true);
        expect(wrapper).toMatchSnapshot();
    });
});