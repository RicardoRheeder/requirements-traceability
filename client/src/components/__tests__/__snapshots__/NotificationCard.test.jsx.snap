// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationCard renders 1`] = `ShallowWrapper {}`;

exports[`NotificationCard renders with children 1`] = `ReactWrapper {}`;
